v8/gruberwine/engine

```diff
@@ -0,0 +1 @@
+jinja
```

Comment:
The `engine` and `parent` files are not needed for current Nikola versions.

Reply:
Removed. The documentation still says they're needed.

Reply:
That README is a bit outdated, I fixed that.